docs/rules/dynamic-import-chunkname.md

```diff
@@ -0,0 +1,66 @@
+# dynamic imports require a leading comment with a webpackChunkName (dynamic-import-chunkname)
+
+This rule reports any dynamic imports without a webpackChunkName specified in a leading block comment in the proper format.
+
+This is a useful rule because if the webpackChunkName is not defined in a dynamic import, Webpack will autogenerate the chunk name.
```

Comment:
This would be better if it explained why it is better to have control over the chunk name vs letting webpack autogenerate one. Basically mention that autogenerated chunk names are not stable across builds, which prevents long-term browser caching.

nit: "webpack" should be lowercase everywhere